### PR TITLE
Volume streaming camera target (without caching)

### PR DESCRIPTION
**Pull request by midlik**

### CODE REVIEW

File: src/mol-plugin/behavior/dynamic/volume-streaming/behavior.ts [hunk 1]
```diff
@@ -303,6 +313,18 @@ export namespace VolumeStreaming {
             }
         }
 
+        private isCameraTargetSame(a?: Camera.Snapshot, b?: Camera.Snapshot): boolean {
+            if (!a || !b) return false;
+            const targetSame = arrayEqual(a.target, b.target);
```
Comment (dsehnal) [inline src/mol-plugin/behavior/dynamic/volume-streaming/behavior.ts:318]:
```suggestion
            const targetSame = Vec3.equals(a.target, b.target);
```

**Review by dsehnal — COMMENTED** [review]
Thanks for the PR @midlik

Before we merge this in addition to addressing the comments, can you please:

- Add your name and update dates of headers in the files you've modified
- Add your name to contributors in package.json
- Add a note to [Unreleased] section of changelog.md

**Comment by midlik** [comment]
Thank you for review. 
I believe I have now addressed all of your comments.

**Review by dsehnal — APPROVED** [review]
Changes look good. Please resolve the merge conflicts and we can merge this.

**Pull request merged**